spec/components/guardian_spec.rb

```diff
@@ -3144,9 +3167,10 @@
       expect(Guardian.new(moderator).can_see_groups?([group])).to eq(true)
       expect(Guardian.new(owner).can_see_groups?([group])).to eq(true)
       expect(Guardian.new.can_see_groups?([group])).to eq(false)
+      expect(Guardian.new(another_user).can_see_groups?([group])).to eq(false)
     end
 
-    it 'correctly handles member visibile groups' do
```

Comment:
Oh my this typo is in so many places. Just goes to show how much code is copied and pasted :)